### ISSUE TITLE
Parse error for skript-mirror code on startup

### ISSUE THREAD

**Issue by RiverThrough** [SkriptLang/Skript#1725]
### Description
Upon updating from 2.3 beta-2 to 2.3 beta-4 I noticed my script file named !!!.sk was failing to load on startup, i have custom skript mirror effects and expressions in that script file so at first I thought maybe it was some of my code that was causing issues even though it worked fine on 2.3 beta-2 so I decided to take all my code out except for the following:

```
import:
        org.bukkit.SkullType
        org.bukkit.Material
        org.bukkit.inventory.ItemStack
        org.bukkit.Bukkit
        java.util.UUID
```
And just with that code above I still got startup errors about the script failing to load, I should note that my full script with my custom effects and expressions reloads just fine without any form of errors after startup but its just failing to parse successfully on startup

### Steps to Reproduce
Have a script with skript-mirror code

https://pastebin.com/vJ2mYGZ8

### Server Information
* **Server version/platform:**      Minecraft 1.13.2 Spigot
* **Skript version:**                2.3 beta-4

**Comment by TheBentoBox** [comment]
I'm also getting this exact thing as well and I'm not sure if it's a Skript or skript-mirror issue but I'm leaning towards Skript since it started happening in recent betas without updating the add-on at all. Needs discussion.

**Comment by Blueyescat** [comment]
I also updated to beta4 and i use many imports, but i didn't get any error 🤔 skript-mirror's weird problems with log handlers

**Comment by RiverThrough** [comment]
> 
> 
> I also updated to beta4 and i use many imports, but i didn't get any error 🤔 skript-mirror's weird problems with log handlers

That is odd, I went ahead and tested it on a fresh server with only skript and skript-mirror installed and only the imports code I provided and I still got the startup errors, I also made sure to update my spigot to the latest version as I believe I was a couple patches behind, I changed the script file name in case it was that but no luck, not sure why you aren't having the errors

**Comment by Snow-Pyon** [comment]
Marking it as bug since I can reproduce it and we're the one who probably broke it somehow. Though, if this happen to be due to skript-mirrror's log handling, I may just close the issue.

**Comment by TheBentoBox** [comment]
I'll also note that when this happens my first handful of scripts can't recognize custom skript-mirror effects, but only those few. Any scripts after the first five or so are able to reference custom effects without errors, but ones in those first few scripts to load fail on most custom effects/expressions. Reloading them works fine but doing `/sk reload scripts` to load all of them again fails in exactly the same way.

**Comment by Snow-Pyon** [comment]
can someone verify in which beta this started happening?

**Comment by Snow-Pyon** [comment]
Added the needs more info label since I can't reproduce it anymore after testing with all betas, but I know it happens, at least that was the case for me yesterday and for many users. If someone can provide a solid test case, I'll be able to find what the actual issue is.

**Comment by RiverThrough** [comment]
> 
> 
> Added the needs more info label since I can't reproduce it anymore after testing with all betas. I know it happens, at least that was the case for me yesterday and for many users. If someone can provide a solid test case, I'll be able to find what the actual issue is.
> 
> For now, you can try restarting your server until it gets fixed, that somehow fixed it for me.

It definitely did not happen on beta-2 (Went ahead and confirmed this now just to be sure), from my tests it happens on beta-3 and beta-4, my only successful test was on my first start attempt after reading your comment about how it randomly fixed, so after that successful first test I shutdown the server and started it backup and sure enough the error was there on startup again, I went through about 10 more restarts and the error was there consistently so I don't understand why it worked with no error on the first start

**Comment by TheBentoBox** [comment]
Same old skript-mirror issue, fixed by updating to a 2.0 alpha of skript-mirror.
